src/docker/02-console/docker.sh

```diff
@@ -31,4 +18,4 @@ if [ -e /var/lib/rancher/conf/docker ]; then
     source /var/lib/rancher/conf/docker
 fi
 
-exec $ARGS $DOCKER_OPTS >/var/log/docker.log 2>&1
+exec /usr/bin/dockerlaunch /usr/bin/docker $ARGS $DOCKER_OPTS # >/var/log/docker.log 2>&1
```

Comment:
I've set stdout and stderr on the respawned process to the console container's ones, so it's visible in regular service logs